### ISSUE TITLE
BTC Deposit shows ALMOST DONE

### ISSUE THREAD

**Issue by hostsleek** [InfraexDev/peatio#81]
Hi All. And my Great Muhammed

Currently solved withdrawal and deposit issues for BTC. But i notice old withdrawal still shows almost done and i am unable to send it back to processing, approve or reject the order. The funds are locked. 

Is this normal

**Comment by nantestdevoloper** [comment]
yes the same problem with me.

withdrawal deposit showing Almost don status not able to recover the btc

**Comment by muhammednagy** [comment]
hello
are you using the updated version here?
did u restart the daemons after pulling?

**Comment by hostsleek** [comment]
Yes i am...  What do you mean by Pulling.. I thought its supposed to be automated.

**Comment by muhammednagy** [comment]
i mean pull the code from github

**Comment by nantestdevoloper** [comment]
no. I have not taken pull.

**Comment by nantestdevoloper** [comment]
using stable version I am able to deposit BTC successfully in account

**Comment by muhammednagy** [comment]
oh you're using master?
we use stable :D

**Comment by hostsleek** [comment]
@nantestdevoloper is this resolved for you. Are your **Almost Done Transaction** cleared now?

**Comment by nantestdevoloper** [comment]
No still not able to recover my BTC.
 @muhammednagy master code

**Comment by muhammednagy** [comment]
@nantestdevoloper you have the balance in your wallet but not showing up on the exchange correct?
